packages/app/non-translatable-markup/src/methods/isAttemptToEditNonTranslatableContent.ts

```diff
@@ -9,9 +8,8 @@ export const isAttemptToEditNonTranslatableContent = (
   text: string,
   updatedText: string,
 ): boolean => {
-  if (!isTextTranslatable(updatedText)) {
-    return true
-  }
+  // early return if the text is the same
+  if (text === updatedText) return false
```

Comment:
`isTextTranslatable` can't be used here now, but looking at the method I think that it is fine to stay without it and added a new condition which is a better early return

Reply:
is the inversion from false to true intended here?

Reply:
Yep, is `text` and `updatedText` is the same it means that user is not trying to edit non translatable content